desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -95,13 +95,62 @@ def parse_events(request, event):
 
 
 def handle_on_app_mention(host_domain, channel_id, user_id, text):
+  if text and 'help' in text:
+    msg_block = help_message(user_id)
+    _send_message(channel_id, block_element=msg_block)
+
   if text and 'queries' in text:
     slack_user = check_slack_user_permission(host_domain, user_id)
     user = get_user(channel_id, slack_user)
 
     handle_query_bank(channel_id, user_id)
 
-  
+
+def help_message(user_id):
+  help_message = [
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "Hey <@{user}> :wave: I'm your SQL Assistant! I'm here to assist users with their SQL queries and much more.\nHere are the few things I can help you with:".format(user=user_id)
+      }
+    },
+		{
+			"type": "divider"
+		},
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:link: Rich preview for Query/Gist links:* Share query/gist links in channel which unfurls in a rich preview, showing query details and result in message thread if available."
+      }
+    },
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:page_facing_up: Share directly from Editor:* Create a query gist, select channel and share your query directly from the Hue Editor window. Requires `share_from_editor` flag enabled."
+      }
+    },
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:eyes: Auto-detect SELECT statements:* Detects SQL SELECT statements in the channel added and suggests a gist link for improved query discussions."
+      }
+    },
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:question: Surfacing Query bank:* Type `@Hue Bot queries` to explore a list of important queries from the latest query bank."
```

Comment:
No need to have 'Bot'?

Reply:
FIxed and will fix in the latest app manifest too!